File: app/src/main/java/com/arashabd/coffeecraftapp/room/entity/Serving.java
package com.arashabd.coffeecraftapp.room.entity;

import androidx.room.ColumnInfo;
import androidx.room.Entity;

@Entity
public class Serving {

    @ColumnInfo(name = "title")
    private String title;

    @ColumnInfo (name = "Description")
    private String Description;

    @ColumnInfo (name = "imageURL")
    private String imageURL;
}

File: app/src/main/java/com/arash/coffeecraftapp/models/Serving.kt
package com.arash.coffeecraftapp.models

import com.google.gson.annotations.SerializedName


data class Serving(

    @SerializedName("title") var title: String? = null,
    @SerializedName("Description") var Description: String? = null,
    @SerializedName("imageURL") var imageURL: String? = null

)
